kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/check/consistency/algorithm/DataMatchDataConsistencyCalculateAlgorithm.java

```diff
@@ -106,11 +106,11 @@ public Optional<DataConsistencyCalculatedResult> calculateChunk(final DataConsis
                 ShardingSpherePreconditions.checkState(!isCanceling(), () -> new PipelineTableDataConsistencyCheckLoadingFailedException(param.getSchemaName(), param.getLogicTableName()));
                 ResultSetMetaData resultSetMetaData = resultSet.getMetaData();
                 int columnCount = resultSetMetaData.getColumnCount();
-                Collection<Object> record = new LinkedList<>();
+                Collection<Object> columnRecord = new LinkedList<>();
```

Comment:
Is there sonar issue of `record`?

Reply:
Yes: https://sonarcloud.io/project/issues?fileUuids=AYePD8WPmlEb3_Pqvjc0&resolved=false&id=apache_shardingsphere&open=AYePD_pgmlEb3_PqvlXe

Reply:
OK